File: php/bateria.php
public class Bateria {
	
	    private $carga;
	   
	    
	    public function Bateria(){
		$this->carga = carga;
		
		}
	
	    public function getCarga() {
	        return $this->carga;
	    }
	    
	    
	    public function descarregando(){
	        if($this->carga>0){
		$this->carga = $this->carga-10;
		else
			echo("Sem Bateria!");
		
	    }
	    public function carregando(){
		if($this->carga!=100){
		$this->carga = $this->carga+5;
		
		else
			echo("Bateria Cheia!");
	       
	        }
	    }   
	     
	 

File: Bateria.java

public class Bateria {

	private int carga;


	public Bateria(int carga){
		
		this.carga = carga;

	}

	public int getCarga() {
		return carga;
	}


	public void descarregar(){
		//se a bateria não estiver vazia descarrega 10%, se não mostra msg sem bateria!
		if(this.carga> 0)
			this.carga = this.carga-10;
		else
			System.out.println("Sem Bateria!");
	}
	public void carregar(){
		//se a bateria não estiver cheia carrega 5% senão mosta msg bateria cheia
		if(this.carga !=100)
			this.carga = this.carga+5;
		else
			System.out.println("Bateria Cheia!");
	}   

}





